### ISSUE TITLE
Completion % incorrect

### ISSUE THREAD

**Issue by ion-alecsandri** [NRGI/rgi-assessment-tool#452]
when log in as researcher, then you complete the answer of assessment.
At this time completion % is not binding for view.
Please check below screenshots.
![1](https://cloud.githubusercontent.com/assets/15853986/15185427/c6ab1816-17a1-11e6-980f-562645f7892e.png)
![2](https://cloud.githubusercontent.com/assets/15853986/15185465/fb4ee476-17a1-11e6-93d4-199d9b57b501.png)

**Comment by alexander-elgin** [comment]
@ion-alecsandri Please, can you provide detailed steps how to reproduce the issue?

**Comment by ion-alecsandri** [comment]
Please go to "Australia 2020 Diamond Mining Assessment" and test it now.

**Comment by alexander-elgin** [comment]
@ion-alecsandri Once again: please, provide detailed steps how I can reproduce the issue. You can start from the user credentials and assessment URL. I cannot reproduce it using the current description.

**Comment by ion-alecsandri** [comment]
Please test "Argentina 2017 Diamond Mining Assessment" now.
it shows that completion % is 0%.
But there are 4 completed answers.
And also you can complete any answer, but completion % does not display correct %.

**Comment by alexander-elgin** [comment]
@ion-alecsandri If I wrote anything unclear, let me know, please. But I am asking you at the third time: please provide the following data to reproduce the issue:
1. User credentials (username and password)
2. Assessment URL
3. Detailed descriptions of the steps that I should perform to get the incorrect percentage of completed answers.

Can you do that?

**Comment by ion-alecsandri** [comment]
1. log in as "ionresearcher, ahsus"
   2.go to "http://rgi-staging.nrgi-assessment.org/assessments/AR-2017-MI"

you can see all of 5 questions are complete, but complete % is 0.

Status: Review Researcher
Last edited by researcher on 05/11/2016 @ 7:22PM
Assessment length: 5
Completed: 0 (0%)
Flagged: 0
Approved: 0
Completed: 5
Unresolved: 0
Unfinalized: 5

**Comment by alexander-elgin** [comment]
@ion-alecsandri How did you do that? Can you describe?

**Comment by ion-alecsandri** [comment]
![1](https://cloud.githubusercontent.com/assets/15853986/15188516/d6e11822-17ae-11e6-84cd-2be51dcb57f3.png)

can you see the screenshot?

**Comment by alexander-elgin** [comment]
@ion-alecsandri Sure, I see the screenshots. I see the probable mismatch as well. And this is the fifth time I am asking you to provide a detailed description of the steps you performed to come from the picture no. 1 (with correct values) to the picture no. 2. Do you have a problem with it?

**Comment by ion-alecsandri** [comment]
1.log in as "ionresearcher, ahsus"
2.
![2](https://cloud.githubusercontent.com/assets/15853986/15189012/3cd180ac-17b1-11e6-93d0-69e9e38f7189.png)
3.
![3](https://cloud.githubusercontent.com/assets/15853986/15189042/6428b92c-17b1-11e6-93b7-525b2f557990.png)

4.then you can see the screenshot
![1](https://cloud.githubusercontent.com/assets/15853986/15189056/71394c12-17b1-11e6-98a9-ed71fa17911c.png)

Here, you can review answers and complete them, and return before page,.

and the compplete % display correct %.

**Comment by alexander-elgin** [comment]
@ion-alecsandri That was really funny. I asked you 5 times to provide a detailed description of the steps how you came to this bug. You showed me the final result only providing a puzzle "Try to guess how I did it". After some time spending on the issue investigation I figured out by myself how to reproduce the issue.
1. A researcher submits assessment for review
2. The supervisor sends the assessment back to the researcher for review i.e. the assessment status become `Review Researcher`
3. The researcher changes (resubmits) any answer in the assessment

After that the `Completed` number (at the same line with the percentage) is decreased by 1.
In this case the number of approved answers only is taken into account for completion percentage. See the image below
![researcher-review](https://cloud.githubusercontent.com/assets/1811689/15203496/3b01d374-1826-11e6-81d1-cee6d9a4de1f.png)
@byndcivilization Is it a BUG?
